openpype/hosts/aftereffects/plugins/create/create_render.py

```diff
@@ -102,6 +132,22 @@ def get_pre_create_attr_defs(self):
     def get_detail_description(self):
         return """Creator for Render instances"""
 
+    def get_dynamic_data(self, variant, task_name, asset_doc,
+                         project_name, host_name, instance):
+        dynamic_data = super(RenderCreator, self).get_dynamic_data(
+            variant, task_name, asset_doc, project_name, host_name,
+            instance
+        )
+
+        if instance is not None:
+            composition_name = instance.get("composition_name")
+            if composition_name:
```

Comment:
I guess it's on purpose to not fill `"composition"` key at all if instance does not have it? (all old instances without `"composition"` will require recreate?).

Reply:
Yes, old instances need to be recreated. I think it is easier than renaming them.